File: EmailVerifychecker/validEmailChecker.py
import re

def is_valid_email(email):
    pattern = r'^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
    return re.match(pattern, email) is not None

email = input("Enter email: ")

if is_valid_email(email):
    print("Valid Email ")
else:
    print("Invalid Email ")

File: EmailVerifychecker/validEmailChecker.cpp
#include<iostream>
#include<regex>
using namespace std;
bool is_validEmail(string email)
{
    regex pattern(R"(^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$)");
    return regex_match(email,pattern);
}

int main()
{
    string email;
    cout<<"enter email address :";
    cin>>email;
    if(is_validEmail(email))
    {
        cout<<"valid email address";
    
        
    }
    else{
        cout<<"invalid email address";
    }


}